### PR TITLE
Add W&B Logging

### PR DESCRIPTION
**Pull request by SauravMaheshkar**
This PR aims to add basic [**Weights and Biases**](https://wandb.ai/site) Metric Logging by appending to the existing training scripts with minimal changes. 

The PR adds 3 extra arguments namely `--use_wandb`, `--wandb_project` and `--wandb_entity` which can be used to specify whether to use wandb, the name of the project to be used (`"nerf-pytorch"` by default) and name of the entity to be used.

### CODE REVIEW

**Comment by SauravMaheshkar** [comment]
Request for Review @yenchenlin

**Comment by yenchenlin** [comment]
Hello @SauravMaheshkar I don't plan to make wandb a default dependency of this project despite it's an awesome tool for logging!
